lib/conduit_lwt_server.mli

```diff
@@ -1,6 +1,8 @@
 
 val close : 'a Lwt_io.channel * 'b Lwt_io.channel -> unit Lwt.t
 
+val set_max_active : int -> unit
```

Comment:
It would be better to leave this decision up to the user of conduit, you might want to have a global limit, per server limit or some combination of them. I went with a global limit here because the underlying resource (process file descriptors) are a global resource too (unless you're running Mirage I guess).
How about  an optional callback argument for `val serve` instead, which would get called with the number of active connections on the current server, the number of active connections globally, and decides what to do with the connection  (accept it, or sleep until either of these two changes).